src/main/java/org/opensearch/eval/samplers/AbstractSamplerParameters.java

```diff
@@ -44,4 +49,12 @@ public String getSampler() {
         return sampler;
     }
 
+    public String getApplication() {
```

Comment:
I'm not sure we need `application` and `search_config` in the sampling part of the eval app.

I think we want them associated with a *query set run* rather than a *query set*. We set them in the query set run config file and use them when analyzing query set runs (=the metrics calculated by the runs).

Or am I missing a point here?

Reply:
No, you're right. Pushed up changes.